ui/src/pages/network/Clearing.tsx

```diff
@@ -57,16 +57,19 @@ export const ClearingServiceTable: React.FC<Props> = ({ services }) => {
     }
   };
 
+  // setClearingAccountName is never used, but clearingAccountName is. How does this work?
+  // eslint-disable-next-line
   const [clearingAccountName, setClearingAccountName] = useState('');
+
+  // setMarginAccountName is never used, but marginAccountName is. How does this work?
+  // eslint-disable-next-line
   const [marginAccountName, setMarginAccountName] = useState('');
```

Comment:
yeah i couldn't figure this one out... a real head scratcher!

Reply:
I think @alexgraham-da last worked on this if i recall right

Reply:
I see the issue...I changed accept offer for `Offer` to use `ClearingOfferModal`, but the `acceptOffer` function is still there for `MarketOffer` and I didn't remove the code for regular `Offer`.

If you change `acceptOffer` on line `111` to:

```typescript
const acceptOffer = async (c: CreateEvent<MarketOffer>) => {
  await ledger.exercise(MarketOffer.Accept, c.contractId, {});
};
```

you should be able to get rid of those variables.

Reply:
That works!